File: Code/Websites/DanpheEMR/wwwroot/DanpheApp/src/app/fraction/shared/fraction-calculation.viewmodel.ts
import { FormGroup, Validators, FormBuilder } from '@angular/forms';

export class FractionCalculationViewModel {

    public BilltxnId: number;
    public ItemName: string; 
    public DoctorPercent: number; 
    public InitialPercent: number; 
    public FinalPercent: number;
    public CreatedOn: number; 
    public DoctorName: string;
    public Designation: string; 
    public TotalAmount: number; 
    public IsParentId: number;

}

File: Code/Websites/DanpheEMR/ViewModel/Fraction/FractionCalculationViewModel.cs
using System;

namespace DanpheEMR.ViewModel
{
 public class FractionCalculationViewModel
    {
        public int? BilltxnId { get; set; }
        public string ItemName { get; set; }
        public decimal? DoctorPercent { get; set; }
        public decimal? InitialPercent { get; set; }
        public decimal? FinalPercent { get; set; }
        public DateTime? CreatedOn { get; set; }
        public string DoctorName { get; set; }
        public string Designation { get; set; }
        public decimal? FinalAmount { get; set; }
        public int? IsParentId { get; set; }
        public int? Hierarchy { get; set; }


        //public int FractionCalculationId { get; set; }
        //public int PercentSettingId { get; set; }
        //public int BillTxnItemId { get; set; }
        //public int DoctorId { get; set; }
        //public double IsParentId { get; set; }
        //public int DesignationId { get; set; }
        //public double InitialPercent { get; set; }
        //public double FinalPercent { get; set; }
        //public int CreatedBy { get; set; }
        //public double FinalAmount { get; set; }
        //public DateTime? CreatedOn { get; set; }
        //public int IsActive { get; set; }

    }
}

